marklogic-data-hub-central/ui/e2e/cypress/integration/scenarios.modeling.tsx

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress"/>
+
+import HomePage from "../support/pages/home";
+import ModelingPage from '../support/pages/modeling';
+
+const homePage = new HomePage();
+const modelingPage = new ModelingPage();
+
+describe('Entity Modeling Page', () => {
+
+  //login with valid account
+  beforeEach(() => {
```

Comment:
We also shouldnt login via GUI for every test. Lets use cy.request('POST', '/login', {
      username,
      password
    }) instead to build up state without using browser’s UI